### PR TITLE
make default logger only print to stderr not file

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by hzxuzhonghu** [comment]
Fix https://github.com/kmesh-net/kmesh/issues/835

File: pkg/controller/telemetry/utils.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ import (
 )
 
 var (
-	log            = logger.NewLoggerField("pkg/telemetry")
+	log            = logger.NewLoggerScope("pkg/telemetry")
```
Comment (LiZhenCheng9527) [inline pkg/controller/telemetry/utils.go:33]:
```suggestion
	log            = logger.NewLoggerScope("telemetry")
```

or controller/telemetry

File: pkg/auth/rbac.go [hunk 2]
```diff
@@ -47,7 +47,7 @@ const (
 )
 
 var (
-	log = logger.NewLoggerField("pkg/auth")
+	log = logger.NewLoggerScope("pkg/auth")
```
Comment (YaoZengzeng) [inline pkg/auth/rbac.go:50]:
```suggestion
	log = logger.NewLoggerScope("auth")
```
`pkg` as prefix doens't seem to make sense?

File: pkg/bpf/bpf.go [hunk 3]
```diff
@@ -39,7 +39,7 @@ import (
 )
 
 var (
-	log  = logger.NewLoggerField("pkg/bpf")
+	log  = logger.NewLoggerScope("pkg/bpf")
```
Comment (YaoZengzeng) [inline pkg/bpf/bpf.go:42]:
```suggestion
	log  = logger.NewLoggerScope("bpf")
```

ditto

**Pull request merged**